packages/react-native-win32/src/Libraries/Components/View/ReactNativeViewViewConfig.win32.js

```diff
@@ -360,7 +360,7 @@ const ReactNativeViewConfig = {
     width: true,
     zIndex: true,
     // [Windows
-    acceptsKeyboardFocus: true,
+    acceptsKeyboardFocus: true, // Can be removed once we're on RN 0.65
```

Comment:
`focusable` wasn't added here in the original PR to deprecate acceptsKeyboardFocus 😕.

@FalseLobster @ksiler I remember there was some conversation about adding keyboard focusability to the Text component. This object effectively dictates what properties RN will let be passed to the RCTText native component when using the builtin Text component, and doesn't include `focusable`. It seems we did patch in support for `acceptsKeyboardFocus` at some point though, though I'm not sure how intentional it was.

`focusable` isn't part of Text component typings either right now FWIW. Do we ever see us making upstream changes to make it focusable? This wouldn't have an impact if we had a forked TextWin32 or whatever that talked to a native component directly.

An alternative here would be to just remove acceptsKeyboardFocus and say we never support that property on stock Text (only none or forked).